core-ui/src/components/Predefined/Details/AuthorizationPolicies/Rules.js

```diff
@@ -0,0 +1,138 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+
+import { LayoutPanel } from 'fundamental-react';
+import { LayoutPanelRow } from 'shared/components/LayoutPanelRow/LayoutPanelRow';
+import { GenericList } from 'react-shared';
+import { Tokens } from 'shared/components/Tokens';
+
+const From = ({ from }) => {
+  const { t } = useTranslation();
+  return (
+    <LayoutPanel>
+      <LayoutPanel.Header>
+        <LayoutPanel.Head title={t('authorizationpolicies.headers.from')} />
+      </LayoutPanel.Header>
+      <LayoutPanel.Body>
+        {from.map(cos => {
+          return Object.entries(cos.source).map(([key, value]) => (
+            <Source name={key} value={value} key={key} />
+          ));
+        })}
+      </LayoutPanel.Body>
+    </LayoutPanel>
+  );
+};
+
+const Source = ({ name, value }) => {
+  const { t } = useTranslation();
+
+  return (
+    <LayoutPanel>
+      <LayoutPanel.Header>
+        <LayoutPanel.Head title={t('authorizationpolicies.headers.source')} />
+      </LayoutPanel.Header>
+      <LayoutPanel.Body>
+        <LayoutPanelRow name={name} value={<Tokens tokens={value} />} />
+      </LayoutPanel.Body>
+    </LayoutPanel>
+  );
+};
+
```

Comment:
The `principals` array that is in the `Source` includes Service Accounts names, we should provide deep linking to them.